Lib/datetime.py

```diff
@@ -4,6 +4,8 @@
 time zone and DST data sources.
 """
 
+__all__ = ("date", "datetime", "time", "timedelta", "timezone", "tzinfo", "MINYEAR", "MAXYEAR")
```

Comment:
Please make sure it follows to limit all lines to a maximum of 79 characters.
PEP8: https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
In case this isn't clear, this can easily be fixed by changing it to:

```suggestion
__all__ = ("date", "datetime", "time", "timedelta", "timezone", "tzinfo",
                 "MINYEAR", "MAXYEAR")
```